### ISSUE TITLE
Relax Kruschke uncertainty report warnings

### ISSUE THREAD

**Issue by aaronyan** [Automattic/abacus#650]
I think we should use "potential issues" and "serious issues" more sparingly. An alert that goes off all the time will eventually get ignored/ assumed to be normal (I know I'm guilty of this). 😅

What do we think about not showing "potential issues" if Kruschke uncertainty is high? Here's an example of an experiment (20440) that shows this:

<img width="1265" alt="Screen Shot 2022-01-27 at 10 41 07 AM" src="https://user-images.githubusercontent.com/4505888/151440566-716cf2e6-bfc8-40f9-a116-d0d60762f4aa.png">

<img width="1265" alt="Screen Shot 2022-01-27 at 10 40 50 AM" src="https://user-images.githubusercontent.com/4505888/151440574-41d259b3-d094-4d60-b53c-aef6c76cd2fa.png">

**Comment by jessie-ross** [comment]
I've never seen any of us, including @yanirs, use the Kruschke uncertainty. It is almost always high or very high. Experimenters certainly don't know what to do with it.

I am uncertain as to what value it is adding, and I think we should even consider removing it.

I think I remember @yanirs having a similar sentiment. cc @demetd in case she is familiar.

**Comment by demetd** [comment]
Yep, Yanir and I both agreed that Kruschke warnings are okay to ignore and are likely too strict: https://github.com/Automattic/data-science/pull/798#issuecomment-603441881

I agree with removing it!

**Comment by aaronyan** [comment]
Cool, sounds good to me to remove as well. I'll edit this issue to reflect the slightly new direction.

The [calculation](https://github.com/Automattic/abacus/blob/c2570d0abec0425a3f67530cefa9ffb337fa403e/src/lib/analyses.ts#L513-L514) seems simple to recreate too, and it would be interesting to see in a future analysis what % of a8c's experiments actually reach the precision criteria.

**Comment by aaronyan** [comment]
Note: this is to remove Kruschke uncertainty from **health report**, but we'll need the Kruschke uncertainty for our improved recommendations (690-gh-Automattic/experimentation-platform)

![150405386-421c80f2-36c8-4c32-a59c-112e0a30e703](https://user-images.githubusercontent.com/4505888/152607398-d1d6c674-a0ec-4629-a60b-af660672cb56.png)
.

**Comment by jessie-ross** [comment]
Ah I didn't see it there, I think we should remove it there as well since we are just ignoring it.

Or make it easier to pass - but I think it is better to ignore.

**Comment by aaronyan** [comment]
> Or make it easier to pass - but I think it is better to ignore.

I'd like to keep the Kruschke uncertainty and make it easier to pass. Since we plan on using the Kruschke uncertainty to inform analysis color and overview text, then we'll prevent Abacus from telling experimenters to take action on an analysis too early.

I like that the new recommendations tell experimenters that a metric is trending towards one direction, but I think we should be strict with when we give the green light and say to "deploy a variant".
